src/components/middle/immedia/Immedia.scss

```diff
@@ -0,0 +1,68 @@
+.MainImmedia {
```

Comment:
Css properties usually have a `kebab-case` name structure. I think it would make it more readable if they follow this convention.

Reply:
Done in 664faf9